vue-frontend/src/components/contributions/DesignExplore.vue

```diff
@@ -0,0 +1,337 @@
+          :speed="3000"
+          :autoplay="{
+            delay: 2000,
+            disableOnInteraction: false,
+          }"
+          :spaceBetween="20"
+          :navigation="true"
+          :loop="true"
+          :loopedSlides="50"
+          class="swiper-container"
+        >
+          <swiper-slide v-for="(post, index) in data.slice(5, 10)" :key="index">
+            <div
+              class="tw-cursor-pointer image2-content"
+              @click="openBlog(post.link)"
+            >
+              <img
+                :src="post.images[0]"
+                :srcset="`${post.images[0]} 400w,${post.images[1]} 800w`"
+                class="tw-w-fit tw-h-fit tw-object-cover tw-drop-shadow-md tw-rounded-lg"
+                alt="UnitTestImage"
+              />
+            </div>
+          </swiper-slide>
+        </swiper>
+      </div>
+    </div>
+  </div>
+
+  <!-- Desktop UI end -->
+</template>
+
+<script>
+import chatapp from "@/assets/images/contributions/explore/1-400w.webp";
+import chatapp800w from "@/assets/images/contributions/explore/1-800w.webp";
+
+import elearningapp from "@/assets/images/contributions/explore/2-400w.webp";
+import elearningapp800w from "@/assets/images/contributions/explore/2-800w.webp";
+
+import fitnessapp from "@/assets/images/contributions/explore/3-400w.webp";
+import fitnessapp800w from "@/assets/images/contributions/explore/3-800w.webp";
+
+import foodorderingapp from "@/assets/images/contributions/explore/4-400w.webp";
+import foodorderingapp800w from "@/assets/images/contributions/explore/4-800w.webp";
+
+import irctcapp from "@/assets/images/contributions/explore/5-400w.webp";
+import irctcapp800w from "@/assets/images/contributions/explore/5-800w.webp";
+
+import meditationapp from "@/assets/images/contributions/explore/6-400w.webp";
+import meditationapp800w from "@/assets/images/contributions/explore/6-800w.webp";
+
+import moviestreamingapp from "@/assets/images/contributions/explore/7-400w.webp";
+import moviestreamingapp800w from "@/assets/images/contributions/explore/7-800w.webp";
+
+import musicapp from "@/assets/images/contributions/explore/8-400w.webp";
+import musicapp800w from "@/assets/images/contributions/explore/8-800w.webp";
+
+import shoppingapp from "@/assets/images/contributions/explore/9-400w.webp";
+import shoppingapp800w from "@/assets/images/contributions/explore/9-800w.webp";
+
+import socialnetworkingapp from "@/assets/images/contributions/explore/10-400w.webp";
+import socialnetworkingapp800w from "@/assets/images/contributions/explore/10-800w.webp";
+
+import AspectRatio from "@/components/utils/AspectRatio.vue";
+import SwiperCore, { FreeMode, Pagination, Autoplay } from "swiper";
+import { Swiper, SwiperSlide } from "swiper/vue";
+SwiperCore.use([FreeMode, Pagination, Autoplay]);
+
+export default {
+  data() {
+    return {
+      activeIndex: 0,
+      gridAnimation: "animation-paused",
+      gridAnimationReverse: "animation-running",
+      pausedId: 0,
```

Comment:
Hover is not working

Reply:
[screen-capture.webm](https://github.com/canopas/canopas-website/assets/98794636/288fe458-6806-4c90-b5af-064bd3b65c3b)

Reply:
its workinig for me.

Reply:
What about desktop devices?

Reply:
Design is different . I have used swiper for above lg devices.